app/service/deed_api.py

```diff
@@ -42,7 +43,8 @@ def get_address(address_json):
 def get_borrowers_json(ids):
     borrowers = []
     for borrower_id in ids:
-        response = requests.get(DEED_API_BASE_HOST + '/borrower/' + str(borrower_id))
+        endpoint_address = DEED_API_BASE_HOST + '/borrower/' + str(borrower_id)
```

Comment:
for maintainability I would make ```'/borrower/'``` a constant at the top of the class so it can be easily changed

https://www.python.org/dev/peps/pep-0008/#constants